lib/install/config/webpack/configuration.js

```diff
@@ -24,7 +24,8 @@ function formatPublicPath(host = '', path = '') {
 
 const output = {
   path: resolve('public', settings.public_output_path),
-  publicPath: formatPublicPath(env.ASSET_HOST, settings.public_output_path)
+  publicPath: `/${settings.public_output_path}/`.replace(/([^:]\/)\/+/g, '$1'),
```

Comment:
@justin808 I am going to push some cleanup for this whole webpack part in another PR. Basically, the idea was to not end up with `//` if `public_output_path` is set to `''` i.e. the assets will be outputted at root. Since we are now using proxy, there is no host for `publicPath` except for internal assets like fonts or images, that's what `publicPathWithHost` is used for. 

> On line 32 below, can you confirm that you're using the adding of the source_path to make non-JS asset inclusion easier? Or are there other reasons?

Yes, that's just adding source path to webpack module resolver.